pom.xml

```diff
@@ -0,0 +1,350 @@
+        <version>${jakarta.resource-api.version}</version>
+    </dependency>
+    <dependency>
+        <groupId>jakarta.security.enterprise</groupId>
+        <artifactId>jakarta.security.enterprise-api</artifactId>
+        <version>${jakarta.security.enterprise-api.version}</version>
+    </dependency>
+    <dependency>
+        <groupId>jakarta.servlet</groupId>
+        <artifactId>jakarta.servlet-api</artifactId>
+        <version>${jakarta.servlet-api.version}</version>
+        <scope>provided</scope>
+    </dependency>
+    <dependency>
+        <groupId>jakarta.servlet.jsp</groupId>
+        <artifactId>jakarta.servlet.jsp-api</artifactId>
+        <version>${jsp-api.version}</version>
+        <scope>provided</scope>
+    </dependency>
+    <dependency>
+        <groupId>jakarta.servlet.jsp.jstl</groupId>
+        <artifactId>jakarta.servlet.jsp.jstl-api</artifactId>
+        <version>${jstl-api.version}</version>
+    </dependency>
+    <dependency>
+        <groupId>jakarta.transaction</groupId>
+        <artifactId>jakarta.transaction-api</artifactId>
+        <version>${jakarta.transaction-api.version}</version>
+    </dependency>
+    <dependency>
+        <groupId>jakarta.validation</groupId>
+        <artifactId>jakarta.validation-api</artifactId>
+        <version>${jakarta.validation-api.version}</version>
+    </dependency>
+    <dependency>
+        <groupId>jakarta.websocket</groupId>
+        <artifactId>jakarta.websocket-api</artifactId>
+        <version>${websocket-api.version}</version>
+        <scope>provided</scope>
+    </dependency>
+    <dependency>
+        <groupId>jakarta.ws.rs</groupId>
+        <artifactId>jakarta.ws.rs-api</artifactId>
+        <version>${jakarta.ws.rs-api.version}</version>
+    </dependency>
+    <dependency>
+        <groupId>jakarta.xml.bind</groupId>
+        <artifactId>jakarta.xml.bind-api</artifactId>
+        <version>${jakarta.xml.bind-api.version}</version>
+    </dependency>
+    <dependency>
+        <groupId>com.sun.xml.bind</groupId>
+        <artifactId>jaxb-osgi</artifactId>
+        <version>${jaxb-osgi.version}</version>
+    </dependency>
+    <dependency>
+        <groupId>org.glassfish.metro</groupId>
+        <artifactId>webservices-api</artifactId>
+        <version>${webservices-api.version}</version>
+    </dependency>
+    <dependency>
+        <groupId>org.glassfish.metro</groupId>
+        <artifactId>webservices-api-osgi</artifactId>
+        <version>${webservices-api-osgi.version}</version>
+    </dependency>
+    <dependency>
+        <groupId>org.glassfish.metro</groupId>
+        <artifactId>webservices-osgi</artifactId>
+        <version>${webservices-osgi.version}</version>
+    </dependency>
+    <dependency>
+        <groupId>org.glassfish.metro</groupId>
+        <artifactId>webservices-tools</artifactId>
+        <version>${webservices-tools.version}</version>
+    </dependency>
+  </dependencies>
```

Comment:
Can add ant.jar dependency as some of the ant classes are being used

<dependency>
    <groupId>org.apache.ant</groupId>
    <artifactId>ant</artifactId>
    <version>1.10.11</version>
    </dependency>